src/api/data/CalloutResponseCommentData/index.ts

```diff
@@ -0,0 +1,60 @@
+import { Filters, Paginated } from "@beabee/beabee-common";
+import CalloutResponseComment from "@models/CalloutResponseComment";
+import { SelectQueryBuilder } from "typeorm";
+import { convertContactToData } from "../ContactData";
+import { fetchPaginated } from "../PaginatedData";
+import {
+  GetCalloutResponseCommentData,
+  GetCalloutResponseCommentsQuery
+} from "./interface";
+
+export function convertCommentToData(
+  comment: CalloutResponseComment
+): GetCalloutResponseCommentData {
+  const commentData = {
+    id: comment.id,
+    contact: convertContactToData(comment.contact),
+    createdAt: comment.createdAt,
+    updatedAt: comment.updatedAt,
+    responseId: comment.responseId,
+    text: comment.text
+  };
+
+  return commentData;
+}
+
+const commentFilter: Filters = {
+  responseId: {
+    type: "text"
+  }
```

Comment:
I think it would be worth adding the other possible filters here, I know they aren't being used currently but we put all that effort into making this work with `fetchPaginated` so we might as well use the magic :magic_wand: 

i.e.
```ts
const commentFilters: Filters = {
  responseId: {
    type: "text"
  },
  contact: {
    type: "contact"
  },
  createdAt: {
    type: "date"
  },
  updatedAt: {
    type: "date"
  },
  text: {
    type: "text"
  }
}
```
I haven't checked that but pretty sure it's right